### DISCUSSION TITLE
Error when pressing "Start Farming"

### DISCUSSION THREAD

**Discussion by oskuroCrypto** [Chia-Network/chia-blockchain#6397] [Support]
Hi!, I'm getting this error

https://i.ibb.co/5RwKMTf/chia-Error.png

and installation doesn't create those 2 paths, just this path:

C:\Program Files (x86)\Chia Core\farmer

What can I do?. I obviously run everything as admin, etc.

Thanks!

**Comment by lvcivs** [comment]
That is not the official chia app. Looks like a custom software made by a pool operator. Wrong place to ask for support.

**Comment by BrandtH22** [comment]
@oskuroCrypto , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
